staging/src/k8s.io/client-go/tools/metrics/metrics.go

```diff
@@ -37,21 +47,51 @@ type ResultMetric interface {
 }
 
 var (
+	// ClientCertExpiration is the lifetime of the client certificate. The value measured in seconds since January 1, 1970 UTC.
+	ClientCertExpiration ExpirationMetric = noopExpiration{}
+	// ClientCertRotationAge is the age of a certificate that has just been rotated.
```

Comment:
Won't this be ~0 most of the time?
When it's just rotated, the cert will be vaild from around the same time.

Reply:
Are you talking about ClientCertRotationAge?

If so, yeah it will be an empty stream until a cert is rotated. So a friend kubelet wont populate this metric with anything until its first certificate rotation. For shielded nodes this will be <10 days. Hypthetically for a kubelet that has been around for a long the histogram will observe the data:
[~10days, ~10days, ~10days, ...]

Does that answer your question?